lib/modules/datasource/conda/prefix-dev.ts

```diff
@@ -0,0 +1,114 @@
+import { logger } from '../../../logger';
+import { isNotNullOrUndefined } from '../../../util/array';
+import type { Http } from '../../../util/http';
+import { MaybeTimestamp } from '../../../util/timestamp';
+import type { Release, ReleaseResult } from '../types';
+import { type File, PagedResponseSchema } from './schema/prefix-dev';
+
+const MAX_PREFIX_DEV_GRAPHQL_PAGE = 100;
+
+const query = `
+query search($channel: String!, $package: String!, $page: Int = 0) {
+  package(channelName: $channel, name: $package) {
+    variants(limit: 500, page: $page) {
+      pages
+      page {
+        createdAt
+        version
+        yankedReason
+        urls {
+          url
+          kind
+        }
+      }
+    }
+  }
+}
+`;
+
+export async function getReleases(
+  http: Http,
+  channel: string,
+  packageName: string,
+): Promise<ReleaseResult | null> {
+  logger.debug(
+    { channel, packageName },
+    'lookup package from prefix.dev graphql API',
+  );
+
+  const files = await getPagedResponse(http, query, {
+    channel,
+    package: packageName,
+  });
+
+  if (!files.length) {
+    return null;
+  }
+
+  let homepage: string | undefined = undefined;
+  let sourceUrl: string | undefined = undefined;
```

Comment:
```suggestion

  let homepage: string | undefined;
  let sourceUrl: string | undefined;
```
undefined is the default value 😉